sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderer.kt

```diff
@@ -182,7 +182,10 @@ internal abstract class MapboxRenderer : MapClient {
 
   @AnyThread
   fun setMaximumFps(fps: Int) {
-    renderThread.setMaximumFps(fps)
+    if (fps <= 0 || fps >= MapView.MAX_POSSIBLE_FPS) {
```

Comment:
```suggestion
    if (fps <= 0 || fps > MapView.MAX_POSSIBLE_FPS) {
```

Reply:
don't see difference, added warning log messages for those situations